lib/helpers.js

```diff
@@ -188,6 +188,39 @@ export function deepEqual(obj1, obj2) {
   return true;
 }
 
+export function createVerifyOptions({
+  verifyPresentationOptions,
+  expectedChallenge,
+  verifiablePresentationRequest,
+  presentation,
+  domain,
+  checks
+}) {
+  // start with verifyPresentationOptions, then overwrite as needed
+  const options = {...verifyPresentationOptions};
+  // update 'checks' with anything additional from 'verifyPresentationOptions'
+  if(verifyPresentationOptions.checks !== undefined) {
+    for(const check in verifyPresentationOptions.checks) {
+      if(verifyPresentationOptions.checks[check] == true) {
+        if(!checks.includes(check)) {
+          checks.push(check);
+        }
+      }
+    }
+  }
+  options.checks = checks;
```

Comment:
```suggestion
const checksSet = new Set(checks);

if(verifyPresentationOptions.checks) {
  for(const [check, enabled] of Object.entries(verifyPresentationOptions.checks)) {
    if(enabled) {
      checksSet.add(check);
    }
  }
}

options.checks = Array.from(checksSet);
```

I'm not sure what's being checked with `verifyPresentationOptions.checks[check]`.